app/addons/documents/views.js

```diff
@@ -18,14 +18,16 @@ define([
   "addons/databases/resources",
 
   // Views
-  "addons/documents/views-queryoptions",
+  //"addons/documents/views-queryoptions",
```

Comment:
please remove the commented code

does it mean we can delete `addons/documents/views-queryoptions.js`?